iOS/Flows/Conversation/Cells/ConversationMessagesCell.swift

```diff
@@ -65,6 +65,7 @@ class ConversationMessagesCell: UICollectionViewCell, ConversationUIStateSettabl
     override init(frame: CGRect) {
         super.init(frame: frame)
 
+        self.configureCollectionLayout(for: .read)
```

Comment:
Now it's the responsibility of the class using the layout to sets the appropriate spacing and positioning parameters.